cmake_support/template_portaudio.def

```diff
@@ -1,61 +0,0 @@
-; $Id: $
```

Comment:
Please do not remove this file.

Reply:
Please respond on #285 where @RossBencina asked to continue discussing this.

Reply:
Based on #285, let's not remove this file for now.